exp/web-dashboard.md

```diff
@@ -0,0 +1,10 @@
+The Anbox Cloud dashboard offers a web interface to create, manage and stream applications. The dashboard is especially helpful for developers new to Anbox Cloud and helps them by offering a simpler and intuitive management interface.
+
+The dashboard may seem more oriented towards developers than end users. However, it only uses available APIs and SDKs and can be a good example to build your own web-based platform.
+
+The dashboard comes pre-installed when you deploy the full version of Anbox Cloud (along with the streaming stack) or the Anbox Cloud appliance. It operates from behind a reverse proxy for performance and security reasons. The dashboard relies on OAuth for user authentication. The only OAuth provider supported right now is [Ubuntu One](https://login.ubuntu.com/). Future versions of Anbox Cloud will allow using other OAuth providers.
```

Comment:
> behind a reverse proxy

Is that true even for the appliance?

Reply:
Yes, as confirmed [here](https://chat.canonical.com/canonical/pl/xwpzd8p14pbz8f4pagm44rrkea)